cmd/operator-sdk/build/cmd.go

```diff
@@ -127,17 +128,21 @@ func buildFunc(cmd *cobra.Command, args []string) error {
 		}
 	}
 
```

Comment:
We need to add the flag with its description to the command and check if the specific `-skip-image` flag was used instead of a number of args > 0